checks.d/openstack.py

```diff
@@ -346,13 +371,31 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Mapping of Nova-managed servers to tags
         self.external_host_tags = {}
 
+        # Set proxy settings
+        self.proxies = {
```

Comment:
Maybe not for this PR, but we should move this boilerplate code upper in the hierarchy or in a free function to avoid duplication and ease maintenance